sample-apps/react-native/dogfood/src/components/ParticipantLayoutButton.tsx

```diff
@@ -0,0 +1,129 @@
+import React, { useState } from 'react';
+import { Pressable, Text, Modal, StyleSheet, View } from 'react-native';
+import GridIconSvg from '../assets/GridIconSvg';
+import { appTheme } from '../theme';
+import { CallParticipantsViewProps } from '@stream-io/video-react-native-sdk';
+
+export type Mode = NonNullable<CallParticipantsViewProps['mode']>;
```

Comment:
no need to export the type Mode also..

again in any screen  ` Mode = React.ComponentProps<CallParticipantsView>["mode"] can be used`